src/hooks/useMenu.ts

```diff
@@ -93,13 +94,16 @@ const menuReducer = (state: IMenuState, action: IMenuActions) => {
       let isMenuOpen = true;
 
       if(state.isMenuPinned) {
+        localStorage.setItem('isMenuOpen', 'false');
         isMenuOpen = false;
+      } else {
+        localStorage.setItem('isMenuOpen', 'true');
```

Comment:
Here Local storage isMenuOpen is global and if changed by one application then it will be automatically changed in another application.
Please add host name (window.location.hostname) in local storage name so it can be unique storage for all applications